apps/site/assets/ts/__v3api.d.ts

```diff
@@ -267,6 +275,7 @@ export interface Schedule {
   stop_sequence: number;
   pickup_type: number;
   train_number?: string;
+  prediction: StopPrediction;
```

Comment:
I see - so you suggest naming the keys - at the very least - as to what type of object they contain. Like `stop_prediction: StopPrediction`?

Reply:
That'd go some way towards making this less confusing, I think. It might result in some unwieldy names when it comes to things like `predicted_or_scheduled_time`, so, follow your heart? (I'm not sure if renaming some of these things would really be in-scope here)